app/models/curator/collection.rb

```diff
@@ -9,8 +9,8 @@ class Collection < ApplicationRecord
 
     belongs_to :institution, inverse_of: :collections, class_name: Curator.institution_class_name
 
-    has_many :admin_set_objects, inverse_of: :admin_set, class_name: Curator.digital_object_class_name, foreign_key: :admin_set_id
+    has_many :admin_set_objects, inverse_of: :admin_set, class_name: Curator.digital_object_class_name, foreign_key: :admin_set_id, dependent: :destroy
 
-    has_many :collection_members, inverse_of: :collection, class_name: Curator.mappings.collection_member_class_name
+    has_many :collection_members, inverse_of: :collection, class_name: Curator.mappings.collection_member_class_name, dependent: :destroy
```

Comment:
I'm not sure we want to use `dependent: :destroy` here? A `DigitalObject` can belong to multiple `Collection`s, and so if we delete a `Collection`, we may not want to delete its associated `DigitalObject`s, since they may belong to other `Collection`s.

Reply:
This is just destroying the `Mappings::CollectionMember` object. It would only `destroy` the associated `digital`object` if `dependent:destroy` was declared in the `belongs_to :digital_object` association on the `Mappings::CollectionMember` class